Sources/KSCrashRecordingCore/KSCrashMonitor.c

```diff
@@ -170,6 +184,8 @@ bool kscm_activateMonitors(void)
         KSLOG_DEBUG("Async-safe environment detected. Masking out unsafe monitors.");
     }
 
+    regenerateEventIds();
```

Comment:
Should we put this inside the lock here?

Reply:
Yup, should be inside the lock.